crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -825,7 +827,7 @@ impl SpecializedMeshPipeline for MeshPipeline {
             shader_defs.push("ENVIRONMENT_MAP".into());
         }
 
-        if key.contains(MeshPipelineKey::IRRADIANCE_VOLUME) {
+        if key.contains(MeshPipelineKey::IRRADIANCE_VOLUME) && IRRADIANCE_VOLUMES_ARE_USABLE {
             shader_defs.push("IRRADIANCE_VOLUME".into());
```

Comment:
Why not just reuse this shader_def instead of using a separate one?

Reply:
I think there are two reasons:

1. MeshPipelineKey::IRRADIANCE_VOLUME is currently getting set in scenes like `deferred_rendering.rs`, even when irradiance volumes aren't used.
2. IRRADIANCE_VOLUMES_ARE_USABLE is used in a few places unconditionally outside of the context of shader defs.

Reply:
I think for now we merge as-is. This isn't hurting anything and we're likely to improve this situation in the future.